systems/Kepler-13.xml

```diff
@@ -52,6 +55,9 @@
 			</planet>
 		</star>
 		<binary>
+			<name>Kepler-13 BC</name>
```

Comment:
I don't think we ever named binaries. I don't see a reason why not to, but it might be of limited use.